lazer/contracts/solana/package.json

```diff
@@ -6,6 +6,7 @@
     "fix:format": "prettier --write **/*.*",
     "test:format": "prettier --check **/*.*",
     "test:anchor": "CARGO_TARGET_DIR=\"$PWD/target\" RUSTUP_TOOLCHAIN=nightly-2025-04-15 anchor test",
+    "update-idl": "RUSTUP_TOOLCHAIN=nightly-2025-04-15 anchor build && cp target/types/pyth_lazer_solana_contract.ts ../../sdk/js-solana/src/idl/pyth-lazer-solana-contract.ts && cp target/idl/pyth_lazer_solana_contract.json ../../sdk/js-solana/src/idl/pyth-lazer-solana-contract.json",
```

Comment:
I don't personally like having a task in one package which reaches into and updates another package.  Personally, I'd put this as a `build:idl` sub-task on the `@pythnetwork/pyth-lazer-solana-sdk` package instead, given it's a task that builds files for that package.

If you really want to be pedantic, you leave something like `RUSTUP_TOOLCHAIN=nightly-2025-04-15 anchor build` here, and then add the `cp` commands in `@pythnetwork/pyth-lazer-solana-sdk`, and set up turbo so the latter depends on the former.

Reply:
yeah that makes sense